### ISSUE TITLE
upgrade to angular 7.1

### ISSUE THREAD

**Issue by xmlking** [pqina/ngx-filepond#10]
please support angular 7.1 to avoid following npm warning 
```
npm WARN ngx-filepond@4.1.0 requires a peer of @angular/common@^6.0.2 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-filepond@4.1.0 requires a peer of @angular/core@^6.0.2 but none is installed. You must install peer dependencies yourself.
```

**Comment by rikschennink** [comment]
Maybe I should just remove those peer dependencies, what do you think?

**Comment by edmundojr** [comment]
Related to this: should I use `ngx-filepond` or `angular-filepond` for integration with Angular 6? What would be the difference between them?

**Comment by xmlking** [comment]
Or use range like 
```json
 "peerDependencies": {
    "@angular/core": ">=5.0.0 <8.0.0"
  },
```

**Comment by rikschennink** [comment]
> Related to this: should I use `ngx-filepond` or `angular-filepond` for integration with Angular 6? What would be the difference between them?

The `angular-filepond` repository has been deprecated, please use `ngx-filepond`.

**Comment by rikschennink** [comment]
Just published version `4.1.1` and switched to `>=6.0.0`
